model/task.go

```diff
@@ -1,7 +1,12 @@
 package model
 
 type Task struct {
-	Description      string    `json:"description"`
-	Tags             []string  `json:"tags"`
-	Project			 string    `json:"project"`
+	Description string    `json:"description"`
+	Tags        []string  `json:"tags"`
 }
+// Tasks struct contains all tasks
+type Tasks struct {
```

Comment:
Not used?

Reply:
not yet... maybe I will remove it and bring it back when there is actually a  need for it.